test/e2e/framework/node/wait.go

```diff
@@ -160,6 +160,22 @@ func WaitForNodeSchedulable(ctx context.Context, c clientset.Interface, name str
 	return false
 }
 
+func WaitForNodeHeartbeatAfter(ctx context.Context, c clientset.Interface, name string, after metav1.Time, timeout time.Duration) bool {
+	framework.Logf("Waiting up to %v for node %s to send a heartbeat after %v", timeout, name, after)
+	for start := time.Now(); time.Since(start) < timeout; time.Sleep(poll) {
+		node, err := c.CoreV1().Nodes().Get(ctx, name, metav1.GetOptions{})
+		if err != nil {
+			framework.Logf("Couldn't get node %s", name)
+			continue
+		}
+		if GetNodeHeartbeatTime(node).After(after.Time) {
+			return true
+		}
+	}
```

Comment:
Can you use `gomega.Eventually` here instead of writing your own polling code?

Reply:
OK, I've also converted two other polling to `gomega.Eventually`